utils/check-style/check-duplicate-includes.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why is it deleted, did you intend to move it to docker dir?

Reply:
Actually is was an accident and actual changes are not there..